services-js/commissions-search/src/ResultList.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class ResultList extends Component {
+  render() {
+    var GET_RESULTS;
```

Comment:
Prefer `let` to `var`.

If it’s a local variable, use camelCase.

Reply:
You could also factor this out into a `makeSearchQuery` method to keep `render` focused on the HTML.

Reply:
done